src/components/date-picker/package.json

```diff
@@ -6,10 +6,9 @@
   },
   "dependencies": {
     "@bcrs-shared-components/interfaces": "^1.0.76",
-    "@bcrs-shared-components/mixins": "^1.1.27",
-    "vue": "^2.7.14"
+    "@bcrs-shared-components/mixins": "^1.1.27"
```

Comment:
I think you don't actually import the shared mixins. Line 84 in the component imports the "local mixin" -- a relative path from this component. (`@/mixins` always resolves to the local mixins folder.)

Basically, when this component is used in Storybook, it uses DateMixin in this repo's mixins folder, but when this component is used in an app, it uses DateMixin in the app's mixins folder.

See if you can get this working while importing the shared mixin instead. 🤞

Reply:
Tried importing the shared mixin and the component seems to work fine. I'll include in my next commit.